docs/Developers/Newcomers_Guide.rst

```diff
@@ -313,8 +313,13 @@ where ``myfork`` is the name of your fork that you added at the previous step.
     You can also add a profile picture to your Github account so that
     you can stand out from the crowd!
 
+<<<<<<< HEAD
 Step 5. Creating a ``Pull Request``
 -----------------------------------
+=======
+Step 6. Creating a Pull Request
+-------------------------------
+>>>>>>> Newcomers_Guide.rst: Step 6 out of place bold text
```

Comment:
can i close this PR and open a new one? its getting quite messy giving more errors as i push it.